types/bbob__plugin-helper/index.d.ts

```diff
@@ -0,0 +1,37 @@
+// Type definitions for @bbob/plugin-helper 3.0
+// Project: https://github.com/JiLiZART/bbob
+// Definitions by: shmee <https://github.com/shme-e>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// /*~ If this module has methods, declare them as functions like so.
```

Comment:
please remove that and all other comments, those were template/boilerplate code

Reply:
oops!